docs/dgx-pod.md

```diff
@@ -77,129 +79,130 @@ The DeepOps service container "DGXie" provides DHCP, DNS, and PXE services to th
 
 ### 1. Prepare the provisioning node
 
-To use DeepOps this repository will need to be downloaded onto the administrator's provisioning system. The `setup.sh` script will then install the following software packages:
-
-* Ansible
-* Docker
-* Git
-* ipmitool
-* python-netaddr (required by kubespray)
+To use DeepOps this repository will need to be downloaded onto the administrator's provisioning system. 
 
 1. Download the DeepOps repo onto the provisioning system:
 
    ```sh
-   git clone --recursive https://github.com/NVIDIA/deepops.git
+  
+   git clone --recurse-submodules https://github.com/NVIDIA/deepops.git
    cd deepops
    git submodule update
+
    ```
 
-   > Note: In Git 2.16.2 or later, use `--recurse-submodules` instead of `--recursive`. If you did a non-recursive clone, you can later run `git submodule update --init --recursive` to pull down submodules
+   > Note: If using Git 2.16.2 or earlier (`git --version`), use  `--recursive` instead of `--recurse-submodules`. If you did a non-recursive clone, you can later run `git submodule update --init --recursive` to pull down submodules
+
+   > With an Ubuntu install you don't get "git" and won't be able to clone the DeepOps repository. You can either install git  `apt -y install.git` or run `setup.sh`  from DeepOps with the following command:   `curl -sL git.io/deepops | bash` 
 
-2. Install Ansible and other dependencies (if the below script fails follow the official [Ansible installation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) steps to install version 2.5 or later). For more information on Ansible and why we use it, consult the [Ansible Guide](ANSIBLE.md).
+   > All subsequent commands are run out of the 'deepops' directory
+
+2. Set up your provisioning machine. 
+
+   This will install Ansible and other software on the provisioning machine which will be used to deploy all other software to the cluster. For more information on Ansible and why we use it, consult the [Ansible Guide](ANSIBLE.md). 
 
    ```sh
+   # Install software prerequisites and copy default configuration.  See the script for components installed.  The `setup.sh` script will install the following software packages: Ansible, Docker, Git, ipmitool, python-netaddr (required by kubespray), pip, wget.
+
    ./scripts/setup.sh
+
    ```
 
-3. Copy and version control the configuration files. The `config/` directory is ignored by the deepops git repo. Create a seperate git repo to track local configuration changes.
 
-   ```sh
-   cp -r config.example/ config/
-   cd config/
-   git init .
-   git add .
-   git commit -am 'initial commit' && cd ..
-   ```
+3. Edit the Ansible inventory file.
+
+Ansible uses an inventory which outlines the servers in your cluster. The setup script from the previous step will copy an example inventory configuration to the `config` directory. 
```

Comment:
Please put additional spaces, so that the text is in line with the number (see above for examples).